drivers/egl/egl_manager.cpp

```diff
@@ -34,6 +34,7 @@
 
 #if defined(EGL_STATIC)
 #define KHRONOS_STATIC 1
+#define GLAD_EGL_VERSION_1_5 true
```

Comment:
Shouldn't you also define `GLAD_EGL_VERSION_1_4`? Or it's not checked with `EGL_STATIC`?

Reply:
The only check with `GLAD_EGL_VERSION_1_4` is in `#if defined(GLAD_ENABLED) && !defined(EGL_STATIC)` (loading EGL via GLAD).